From 0e52273d87ebb74f7da776b9744ffca00ff358fd Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Mon, 4 Apr 2011 22:46:32 -0500
Subject: fix-gnu-kfreebsd-startup.diff
 * Emacs should no longer hang on startup on Debian GNU/kFreeBSD.
   Patch: fix-gnu-kfreebsd-startup.diff
   Date: Thu Aug 12 04:21:09 UTC 2010
   Added-by: Rob Browning <rlb@defaultvalue.org>
   Provided-By: Petr Salinger <Petr.Salinger@seznam.cz>
   Bug: 559392
   Status: new

---
 src/s/gnu-kfreebsd.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/src/s/gnu-kfreebsd.h b/src/s/gnu-kfreebsd.h
index a1e8c02..89563de 100644
--- a/src/s/gnu-kfreebsd.h
+++ b/src/s/gnu-kfreebsd.h
@@ -5,6 +5,11 @@
 #undef SYSTEM_TYPE
 #define SYSTEM_TYPE "gnu/kfreebsd" /* All the best software is free */
 
+#undef INTERRUPT_INPUT
+#define BROKEN_SIGIO
+#define BROKEN_SIGURG
+#define BROKEN_SIGPOLL
+
 #define NO_TERMIO               /* use only <termios.h> */
 
 /* arch-tag: 8d098200-2586-469e-99ab-6d092c035e03
-- 
1.7.4.1

